src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -1232,6 +1232,51 @@ describe("Artwork type", () => {
     })
   })
 
+  describe("#shippingContinentalUsOnly", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          shippingContinentalUsOnly
+        }
+      }
+    `
+    it("is true when domestic_shipping_fee_cents and international_shipping_fee_cents reflect continental only shipping", () => {
+      artwork.domestic_shipping_fee_cents = 1000
+      artwork.international_shipping_fee_cents = 0
```

Comment:
If `international_shipping_fee_cents` is 0, doesn't that mean that it has _free_ shipping? I would expect this case to only be true if it is `null`.

Reply:
I'm not sure! I duplicated the logic that `shippingInfo` uses, which has the same check, i.e. checking that `international_shipping_fee_cents` is falsey, not just `null`: https://github.com/javamonn/metaphysics/blob/50ef4974728bcb5363539711c69a98ac1d25149d/src/schema/artwork/__tests__/artwork.test.js#L1186-L1196. Perhaps we should update both spots?

Reply:
Let's check in the #commerce-pilot-pde channel since there may be some nuance to the UI/modeling that I'm not aware of. But if it's supposed to just check for `null` then we should for sure update both!